src/Avalonia.Controls/TreeView.cs

```diff
@@ -76,6 +83,15 @@ public event EventHandler<SelectionChangedEventArgs>? SelectionChanged
             remove => RemoveHandler(SelectingItemsControl.SelectionChangedEvent, value);
         }
 
+        /// <summary>
+        /// Represents the event that is raised when the expansion state of a <see cref="TreeViewItem"/> changes within a <see cref="TreeView"/>.
+        /// </summary>
+        public event EventHandler<TreeViewItemExpansionChangedEventArgs> ExpansionChanged
```

Comment:
For Expander, we use ExpandedEvent/ExpandingEvent and CollapsedEvent/CollapsingEvent. It would be expected to have at least ExpandedEvent and CollapsedEvent here for consistency, but not ExpansionChanged.

Reply:
Yes, let's skip ExpandingEvent and CollapsingEvent events for now, as these are cancellable and more tricky to implement.

ExpandedEvent and CollapsedEvent should be enough for most apps.